include/zephyr/bluetooth/audio/audio.h

```diff
@@ -69,6 +69,80 @@ enum bt_audio_context {
 					  BT_AUDIO_CONTEXT_TYPE_ALERTS | \
 					  BT_AUDIO_CONTEXT_TYPE_EMERGENCY_ALARM)
 
+enum bt_audio_parental_rating {
```

Comment:
Would be good to have at least short doxygen documentation of those values, what are those, where they come from etc.